flet/PyAlgoMate.py

```diff
@@ -10,6 +10,7 @@
 from importlib import import_module
 import flet as ft
 import sys
+import sentry_sdk
```

Comment:
Why not log_setup here?

Reply:
Because logs have been setup in this file from line 23 to 42. If you want to consolidate the logs from this file, then we'll need to remove the code from here and then import `log_setup`